examples/Client/StateManagement/BulkStateExample.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Dapr.Client;
+
+namespace Samples.Client
+{
+    public class BulkStateExample: Example
+    {
+        private static readonly string firstKey = "testKey1";
+        private static readonly string secondKey = "testKey2";
+        private static readonly string firstEtag = "123";
+        private static readonly string secondEtag = "456";
+        private static readonly string storeName = "statestore";
+
+        public override string DisplayName => "Using the State Store";
+        public override async Task RunAsync(CancellationToken cancellationToken)
```

Comment:
Nit: It'd be better to have a space between these two lines.

Reply:
Fixed